pkg/operator/starter.go

```diff
@@ -150,9 +134,30 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	// trusted) should be valid for at least this long.
 	minimumTrustDuration := 395 * 24 * time.Hour
 
-	if featureGates.Enabled(features.FeatureShortCertRotation) {
-		minimumTrustDuration = time.Hour + 15*time.Minute
-		signingCertificateLifetime = time.Hour*2 + 30*time.Minute
+	if ok, err := starter.IsMicroshift(ctx, kubeClient); err == nil && !ok {
```

Comment:
I don't think we need to detect Microshift in the operator because Microshift does not run the operator, just the operand.